src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -1,36 +1,54 @@
 package lotto.domain;
 
+import lotto.strategy.LottoMakeStrategy;
+
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
 public class LottoTicket {
+
+    private static final int LOTTO_NUMBER_FIX_SIZE = 6;
+    private static final String LOTTO_TICKET_NULLPOINT_EXCEPTION = "로또가 생성 되지 않았습니다.";
     private final List<LottoNumber> lottoTicket;
 
     public LottoTicket(){
-        this.lottoTicket = lottoNumbersMake();
+        this(new ArrayList<>());
+    }
+
+    public LottoTicket(LottoMakeStrategy lottoMakeStrategy){
+        this.lottoTicket = lottoMakeStrategy.valueOf();
```

Comment:
```suggestion
        tthis(lottoMakeStrategy.valueOf());
```
주 생성자를 호출해 값을 초기화해야 중복을 제거할 수 있다.